robot-core/src/main/java/org/obolibrary/robot/ExplainOperation.java

```diff
@@ -112,6 +304,38 @@ private static String renderTree(Tree<OWLAxiom> tree, OWLObjectRenderer renderer
     return builder.toString();
   }
 
+  private static String renderAxiomWithImpact(
+      List<OWLAxiom> axioms,
+      int impact,
+      OWLObjectRenderer renderer,
+      Map<OWLAxiom, Set<String>> associatedOntologyIds) {
+    StringBuilder builder = new StringBuilder();
+    builder.append("## Axioms used " + impact + " times" + "\n");
```

Comment:
String concats in StringBuilders should be chained builder calls (better performance):
```
builder.append("## Axiom used ").append(impact).append(" times\n");
```